tools/jira_cmd.py

```diff
@@ -118,9 +120,12 @@ def get_data_for_print(issues, issues_count=None, print_fields=None):
         if 'component' in headers:
             row['component'] = [c.name for c in i.fields.components]
         if 'priority' in headers:
-            row['priority'] = i.fields.priority.name
+            try:
+                row['priority'] = i.fields.priority.name
+            except Exception:
+                row['priority'] = i.fields.priority
```

Comment:
What happens here? and can we make it simpler?

Reply:
In some projects the priority field is an object that has a name property and in others it is not such an object